template/deployments/helm/charts/templates/_helpers.tpl

```diff
@@ -26,16 +26,16 @@ If release name contains chart name it will be used as a full name.
 {{/*
 Create chart name and version as used by the chart label.
 */}}
-{{- define "{{APPNAME}}.chart" -}}
+{{- define "my-app.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
 {{/*
 Common labels
 */}}
-{{- define "{{APPNAME}}.labels" -}}
-helm.sh/chart: {{ include "{{APPNAME}}.chart" . }}
-{{ include "{{APPNAME}}.selectorLabels" . }}
+{{- define "my-app.labels" -}}
```

Comment:
If were going to keep using the helper file, lets prefix these to `helper`:

`helper.labels`
`helper.selectorLabels` 

and so on, and refrence them as such within the manifests

Reply:
I'm not using the helper file, I haven't deleted it yet. No where in the template directory I'm referencing the helper file.